core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -43,6 +45,36 @@
     @ConfigItem(defaultValue = "true")
     public boolean enableJni;
 
+    /**
+     * Defines the user language used for building the native executable.
+     * <p>
+     * Defaults to the system one.
+     */
+    @ConfigItem(defaultValue = "${user.language:}")
```

Comment:
@radcortez is it safe to have an empty default value with a non-Optional property? It seems to work but I'm a bit surprised :).

Reply:
That is definitely a bug :)

I guess we never covered that case. We probably just blindly accept the default value from the expansion without checking if there is something there. I'm going to add a fix for it, so I recommend to not lean on it :)

Reply:
Well, if I merge it before you fix it, it's your turn to fix it :).

I pushed a fix to use `Optional`.

Reply:
Ahahaha! Good job!